kong/plugins/aws-lambda/handler.lua

```diff
@@ -30,18 +31,24 @@ local function get_now()
 end
 
 
+local function initialize()
+  AWS_GLOBAL_CONFIG = aws_config.global
+  AWS = aws()
```

Comment:
Do we need concurrency control on these module top-level variable initialization?

Reply:
There already is code in [lua-resty-aws](https://github.com/Kong/lua-resty-aws/blob/120c6ce3443f010a9f19766b7b7ec2ed9d457ad5/src/resty/aws/utils.lua#L232) to make sure that the region detection is performed only once, so this should be safe.